src/NuGet.Clients/NuGet.Console/ConsoleInitializer.cs

```diff
@@ -45,18 +43,18 @@ private static Task<Action> GetInitializeTask()
                 // which is very rare.
             }
 
-            var initializer = componentModel.GetService<IHostInitializer>();
-            return Task.Factory.StartNew(state =>
+            var initializer = comSvc.GetService<IHostInitializer>();
+            return await System.Threading.Tasks.Task.Factory.StartNew(state =>
```

Comment:
[doc](https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.taskfactory.startnew?view=net-5.0#remarks) recommends `Task.Run` API.

Reply:
Changing to `Task.Run` Api will make it run code in threadpool. But `System.Threading.Tasks.Task.Factory.StartNew`  runs in current synchronization context which could be UI or threadpool, not sure because I can't hit this part of code, it looks this code is no longer in use.  Since there is some unknowns better to leave as it's.  Maybe we can to make it obsolete, currently it's not reference in our code.

Reply:
The old code was not async (it returned a `Task`, but didn't use the `async` modifier).

Since this PR changes that, I don't see the point of keeping the `StartNew`, or using `Task.Run`. It's not clear to me if it's trying to run on the background thread. If so, use `await TaskScheduler.Default` instead.

Reply:
Ok. I removed `StartNew`, but didn't use `TaskScheduler.Default` because previous System.Threading.Tasks.Task.Factory.StartNew runs in current synchronization context so doesn't want to change it.